Makefile

```diff
@@ -21,22 +22,25 @@ in/airports.dat:
 in/countries.dat:
 	wget -c https://raw.githubusercontent.com/jpatokal/openflights/master/data/countries.dat -O in/countries.dat
 
-in/continents.geojson: $(DIRS)
-	zcat continents.geojson.gz > in/continents.geojson
+geojson: continents.geojson.gz
```

Comment:
what does `continents.geojson.gz` rule mean here?

Reply:
dependence on the existence of the file continents.geojson.gz

Reply:
@roman-khimov, is it a good practice to place a dependency as a prerequisite but without any rule to create it? (i do not know)

Reply:
It's OK since this file does exist in the repository. If it changes, the dependency will trigger a rebuild of the target.